packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -62,7 +62,7 @@ export const getCashierValidations = cashier_arr => {
 };
 
 export const excluded_notifications = isMobile()
-    ? ['contract_sold']
+    ? ['contract_sold', 'p2p_completed_order', 'p2p_ad_type_changed']
```

Comment:
Due to notifications being refactored, in order to show p2p related notifications, the notification type must be manually added here